### PR TITLE
fix bug #2656

### PR DESCRIPTION
**Pull request by junhao69535**
Fix https://github.com/unbit/uwsgi/issues/2656

### CODE REVIEW

**Comment by xrmx** [comment]
@methane what do you think?

File: core/master_utils.c [hunk 1]
```diff
@@ -54,6 +54,12 @@ void uwsgi_destroy_processes() {
 
         uwsgi_detach_daemons();
 
+		for (i = 1; i <= uwsgi.numproc; i++) {
+                if (uwsgi.workers[i].pid > 0) {
+                        waitpid(uwsgi.workers[i].pid, &waitpid_status, 0);
+                }
+        }
+
```
Comment (xrmx) [inline core/master_utils.c:62]:
```suggestion
	for (i = 1; i <= uwsgi.numproc; i++) {
		if (uwsgi.workers[i].pid > 0) {
			waitpid(uwsgi.workers[i].pid, &waitpid_status, 0);
		}
	}

```

**Review by methane — APPROVED** [review]
LGTM

**Comment by xrmx** [comment]
Merged in #2671

**Pull request closed without merging**